packages/react-core/src/components/Modal/Modal.tsx

```diff
@@ -38,6 +38,8 @@ export interface ModalProps extends React.HTMLProps<HTMLDivElement> {
   description?: React.ReactNode;
   /** Variant of the modal */
   variant?: 'small' | 'large' | 'default';
+  /** Flag indicating if modal content should have no padding*/
+  noPadding?: boolean;
```

Comment:
I wonder if this prop name is too generic and might overlap or be confusing with other props we could introduce into the modal, like one that removes all padding from the modal, or just the padding from the `<ModalBoxBody>` element. I have seen potential use cases for the former.

Technically this prop removes the `<ModalBoxBody>` element all together. 

Here is what it looks like now with regular modal content and `noPadding`

<img width="728" alt="Screen Shot 2020-05-04 at 3 46 26 PM" src="https://user-images.githubusercontent.com/35148959/81012273-e3c93980-8e1e-11ea-818a-2f7e2898ba2c.png">

Reply:
hm... would it makes sense to have it be called noModalBoxBody? Would that make sense to a consumer? Could something like removeBody make more sense?

Reply:
Yeah, `noModalBoxBody` definitely isn't as easy of a name, though it's a more accurate description of what it does. Or maybe `noModalBoxBodyWrapper` since you can still pass content and it puts it where the body would go. `noModalBoxBody` and `removeBody` could be confused with the modal having no body content. At any rate, I think as long as we document it, any of those names seem OK to me. I'm interested in others' thoughts.

Reply:
I could go with noModalBoxBodyWrapper... it's a mouthful.
@tlabaj @jschuler @dlabrecq any thoughts?